Simcoe/Mixpanel/Mixpanel.swift

```diff
@@ -0,0 +1,178 @@
+//
+//  Mixpanel.swift
+//  Simcoe
+//
+//  Created by Yoseob Lee on 2/28/17.
+//  Copyright © 2017 Prolific Interactive. All rights reserved.
+//
+
+import Mixpanel
+
+typealias MixpanelProperties = [String: MixpanelType]
+
+/// The Mixpanel analytics provider.
+public class MixpanelPlaceholder {
+
+    /// The name of the tracker.
+    public let name = "Mixpanel"
+
+    /// Initializes an API object with the input token.
+    ///
+    /// - Parameters:
+    ///   - token: The token.
+    ///   - launchOptions: The launch options.
+    ///   - flushInterval: The interval for background flushing.
+    ///   - instanceName: The instance name.
+    public init(token: String,
+                launchOptions: [UIApplicationLaunchOptionsKey: Any]? = nil,
+                flushInterval: Double = 60,
+                instanceName: String = UUID().uuidString) {
+        Mixpanel.initialize(token: token,
+                            launchOptions: launchOptions,
+                            flushInterval: flushInterval,
+                            instanceName: instanceName)
+    }
+
+
+    /// Force uploading of queued data to the Mixpanel server.
+    public func stop() {
```

Comment:
Updated to be `flush()`

Haven't added back `stop()` as I am not 100% clear what that functionality would be yet